_projects/2023/100385252/100385252.Rmd

```diff
@@ -0,0 +1,660 @@
+
+## 2. Create Graph
+
+### Initial elements
+
+To begin, the chart is plotted using geom_line() and geom_area(), enabling the representation of life expectancy in stacked areas for each of the years. To match the plot with the original graph's palette, two vectors are created, one containing the specific colors to fulfill the areas, and other to draw the lines.
+
+To adjust the scale of the plot and stick as much as possible to the shape of the original plot, breaks in the axes are manually introduced along with th axis labels. However, later on, we will need to include some annotations in the right side of the plot. Therefore, the x-limits are set beyond 1 to create additional space.
+
+```{r fig.height = 5, fig.width = 7, fig.align = "center"}
+
+area_color <- c("#84A98C", "#FFD399","#FFCAD3") 
+line_color <- c("#238B45", "#ff6900", "#99000d") 
+
+plot <- 
+  ggplot(dataset, 
+         aes(x = share, y = LifeExpectAtBirth,
+             label= as.factor(Year), 
+             color = as.factor(Year))) +
+  geom_line() +
+  geom_area(aes(fill = as.factor(Year)), 
+            position = position_dodge(width = 0), 
+            alpha = 0.8) +
+  scale_fill_manual(values = area_color) +
+  scale_color_manual(values = line_color) +
+  scale_x_continuous(
+    breaks = c(0.0, 0.25, 0.5, 0.75, 1), 
+    labels = c("0", "1/4", "1/2", "3/4", "1"),
+    expand = c(0, 0),
+    limits = c(0, 1.1)) +
+  scale_y_continuous(
+    breaks = seq(0, 90, by = 10), 
+    labels = c("","10","20", "30", "40", "50", "60", "70", "80", "90"),
+    expand=c(0,0),
+    limits = c(0, 95)) +
+  guides(color = FALSE, fill = FALSE)
+
+plot  
+
+```
+
+### Managing labels
+
+Our next step is to include the labels of the countries for each year along the x-axis. To achieve this, we consider the subset formed by those countries with "Yes" in the label column.
+
+Since the plot involves a large number of labels inside the graph, geom_text_repel() is used to adjust the position of the countries and minimize overlap. Another advantage is that it allows for constraining the labels within specified limits, avoiding them from extending beyond the graph.
+
+```{r fig.height = 5, fig.width = 7, fig.align = "center"}
+
+plot <- 
+  plot + 
+  geom_text_repel(data = subset(dataset, label == "Yes"), aes(label = Country),
+                  xlim = c(0, 1),
+                  nudge_y = 0.05,  
+                  max.overlaps = 10, 
+                  colour = "black",
+                  size = 4, 
+                  segment.size = 0, 
+                  segment.color = NA) 
+  
+
+plot
+```
+
+### Appearance adjustments
+
+We proceed by adding the corresponding title, subtitle and caption to the plot, as well as adjusting the graph appearance, where the font used is "Playfair Display", downloadable from Google Fonts.
+
+```{r fig.height = 5, fig.width = 7, fig.align = "center"}
+
+sysfonts::font_add_google("Playfair Display") 
+showtext::showtext_auto() 
```

Comment:
Instead of calling this line, it's better to add `fig.showtext=TRUE` to all chunks with graphic output.